CHANGELOG.md

```diff
@@ -15,4 +15,5 @@
 - Error Module brief design & using httplib module to validate this design. [4453](https://github.com/beego/beego/pull/4453)
 - Fix 4444: panic when 404 not found. [4446](https://github.com/beego/beego/pull/4446)
 - Fix 4435: fix panic when controller dir not found. [4452](https://github.com/beego/beego/pull/4452)
-- Fix 4456: Fix router method expression [4456](https://github.com/beego/beego/pull/4456)
\ No newline at end of file
+- Fix 4456: Fix router method expression [4456](https://github.com/beego/beego/pull/4456)
+- Fix 4060: Fix delete by mistake [4060](https://github.com/beego/beego/issues/4060)
```

Comment:
Fix XXX, xxx should be the issue's number, not PR's number.
I just found out that both 4456 and 4060 are wrong.

Reply:
Sorry, I made mistake

Reply:
Fix 4060: Fix delete by mistake [4460](https://github.com/beego/beego/pull/4460)

Fix [issue number]: summary [pull number](pull link)